docs/components/connectors/out-of-the-box-connectors/email.md

```diff
@@ -0,0 +1,695 @@
+    {
+      "field":"FROM",
+      "value":"example@camunda.com"
+    },
+    {
+      "operator":"OR",
+      "criteria":[
+        {
+          "field":"SUBJECT",
+          "value":"urgent"
+        },
+        {
+          "field":"SUBJECT",
+          "value":"important"
+        }
+      ]
+    }
+  ]
+}
+```
+
+The above query will return emails that are from "example@camunda.com" and have a subject containing either "urgent"
+or "important".
+
+A simpler query without logical operators may look like this:
+
+```json
+{
+  "field":"FROM",
+  "value":"example@camunda.com"
+}
+```
+
+The search functionality currently supports the following logical operators:
+
+- **AND**: Returns emails that match all the specified criteria.
+- **OR**: Returns emails that match any of the specified criteria.
+
+The following email fields can be used to set search criteria:
+
+- **BODY**: The content of the email body.
+- **SUBJECT**: The subject line of the email.
+- **FROM**: The email address of the sender.
+
+When using an operator such as AND or OR, you must also define a criteria array. This array contains the individual
+conditions that the search will evaluate against the emails. Each condition within the criteria array is itself a JSON
+object with a field and a value.
+
+#### Notes
+
+- If an operator is set, the criteria array must also be defined.
+- Each criterion within the criteria array is applied to the specified field based on the value associated with it.
+
+### Example Response
+
+Returned Response:
+
+```json
+[
+  {"messageId":"MessageId", "subject":"Important"},
+  {"messageId":"MessageId2", "subject":"Urgent"}
+]
+```
+
+## Move Email
+
+This task enables users to transfer an email from one folder to another, streamlining inbox organization.
+
+### Parameters
+
+- `MessageId`: The identifier of the email that needs to be moved.
+- `Source folder`: (Optional) The folder from which the email will be moved. If not specified, the default is INBOX.
+- `Target folder`: The destination folder where the email will be placed. To specify a new folder or a nested hierarchy,
+  use a dot-separated path (e.g., 'Archive' or 'Projects.2023.January'). The system will automatically create any
+  non-existing folders in the path.
+
+### Response Structure
+
+Upon successful completion of the move operation, the response will contain a JSON object with the following details:
+
+- `messageId`: The messageId of the email that was moved.
+- `from`: The source folder from which the email was moved.
+- `to`: The target folder to which the email has been moved.
+
+### Example Response
+
+The example below demonstrates the expected JSON response after an email has been successfully moved:
+
+```json
+
+{
+  "messageId":"VE1P191MB1101730EEA31B2FEAB320143919A2@VE1P191MB1101.EURP191.PROD.OUTLOOK.COM",
+  "from":"INBOX",
+  "to":"TEST"
+}
+```
+
+</TabItem>
+
+<TabItem value='inbound'>
+
+# Prerequisites
+
+This inbound connector only work with IMAP server.
```

Comment:
I would highlight this, maybe a warn/note component, you can find examples in other connectors docs